src/components/Dropdown/Dropdown.test.tsx

```diff
@@ -30,10 +29,10 @@ describe('<Dropdown />', () => {
 		const label = 'Show options';
 		const isOpen = false;
 		const children = <div>content item</div>;
-		const dropdown = renderDropdown({ children, label, isOpen });
+		const { container } = renderDropdown({ children, label, isOpen });
 
 		const dropdownContent = await waitFor(() =>
-			documentOf(dropdown).getElementsByClassName('c-menu--visible--default')
+			container.getElementsByClassName('c-menu--visible--default')
```

Comment:
Je kan ook dit doen: `container.querySelector('.c-menu--visible--default')` om 1 element te queryen
en in uw expect kan je dan eenvoudig `expect(dropdownContent).toBeInTheDocument()`